logtail/logtail.go

```diff
@@ -468,10 +506,34 @@ func (l *Logger) encodeText(buf []byte, skipClientTime bool, level int) []byte {
 	b := make([]byte, 0, len(buf)+overhead)
 	b = append(b, '{')
 
-	if !skipClientTime {
-		b = append(b, `"logtail": {"client_time": "`...)
-		b = now.AppendFormat(b, time.RFC3339Nano)
-		b = append(b, "\"}, "...)
+	if includeLogtail {
+		b = append(b, `"logtail": {`...)
+		needsComma := false
```

Comment:
Did this, except I did a subslice more directly -- `TrimRight` docs say that it looks at UTF-8 encoded code points, which seems like it would have more overhead.

Reply:
No need to worry about that. `TrimRight` has been optimized to handle a single-byte cutset pretty efficiently 😄: https://go-review.googlesource.com/c/go/+/323318

Reply:
Also, I think using `TrimRight` is more correct in the rare case where none of the `logtail` fields are outputed, in which case you will accidentally truncate something that is not a `,`.

Reply:
Nice, TIL